src/libawkward/builder/BoolBuilder.cpp

```diff
@@ -59,7 +59,7 @@ namespace awkward {
   BoolBuilder::null() {
     BuilderPtr out = OptionBuilder::fromvalids(options_, shared_from_this());
     out.get()->null();
-    return std::move(out);
+    return out;
```

Comment:
The warning said that the use of `std::move` for a local object prevents copy elision. I don't think copy-vs-move matters much here, since it's a pointer (`BuilderPtr`), not the object itself.